Scripts/Diagnostics/Azure_DataFactory_DP_Avoid_Plaintext_Secrets.ps1

```diff
@@ -0,0 +1,282 @@
+$debug = $true
```

Comment:
General thought, do we need to add required module list and installation steps as well in some common place.

Reply:
removed file